keystore/keystore.go

```diff
@@ -439,17 +439,29 @@ type ServerKeyStore interface {
 	Reset()
 }
 
+// KeyMarker set key market for KeyDescription
+type KeyMarker string
+
+// MarkerCurrent represent current KeyMarker
+const (
+	MarkerCurrent KeyMarker = "current"
+	MarkerRotated           = "rotated"
+)
+
 // KeyDescription describes a key in the keystore.
 //
-// "ID" is unique string that can be used to identify this key set in the keystore.
+// "Idx" is unique integer that can be used to identify key in the context of KeyID.
+// "KeyID" is unique string that can be used to identify this key set in the keystore.
 // "Purpose" is short human-readable description of the key purpose.
 // "ClientID" and "AdditionalContext" are filled in where relevant.
+// "CreationTime" used to display creation time of rotated key
 type KeyDescription struct {
-	ID       string
-	Purpose  KeyPurpose
-	ClientID []byte `json:",omitempty"`
-	// used to display creation time of rotated key
-	CreationTime time.Time
+	Idx          int
```

Comment:
what about to name in "Index" instead of "Idx"? Because all other fields have fullname and this looks like abbreviation for something